qms-tests/sanity/tests/model/documents/document-content-page.ts

```diff
@@ -144,6 +160,40 @@ export class DocumentContentPage extends DocumentCommonPage {
       .fill(title)
   }
 
+  async clickOnAddCategoryButton (): Promise<void> {
+    await this.addCategoryButton.click()
+  }
+
+  async selectControlDocumentSubcategory (
+    buttonName: 'My Document' | 'Library' | 'Templates' | 'Categories'
+  ): Promise<void> {
+    switch (buttonName) {
+      case 'My Document':
+        await this.myDocument.click()
+        break
+      case 'Library':
+        await this.library.click()
+        break
+      case 'Templates':
+        await this.templates.click()
+        break
+      case 'Categories':
+        await this.categories.click()
+        break
+      default:
+        throw new Error('Unknown button')
+    }
+  }
+
+  async fillCategoryForm (categoryTitle: string, description: string, categoryCode: string): Promise<void> {
+    await this.categoryTitle.fill(categoryTitle)
+    await this.description.fill(description)
+    await this.categoryCode.fill(categoryCode)
+    await this.createButton.click()
+    await expect(this.page.getByText(categoryTitle)).toBeVisible()
```

Comment:
Can we separate expects from operations? It doesn't feel right that `fillSmthForm` also checks something. This way it's hard to read.

Reply:
sure we can